packages/web/config/chain-infos.ts

```diff
@@ -1957,6 +1957,13 @@ chainInfos.push({
       coinGeckoId: "polkadot",
       coinImageUrl: "/tokens/dot.svg",
     },
+    {
+      coinDenom: "MATIC",
```

Comment:
this describes MATIC, while the token is still WMATIC. the symbol, minimal denom, cgid, and image name are not correct here.

Reply:
> this describes MATIC, while the token is still WMATIC. the symbol, minimal denom, cgid, and image name are not correct here.

Sorry, resolved.  Had it right in the other file, not sure what I was thinking.